lib/trivial.nix

```
@@ -448,6 +448,39 @@ rec {
   isFunction = f: builtins.isFunction f ||
     (f ? __functor && isFunction (f.__functor f));
 
+  /* Copy the function arguments (__functionArgs) of the first function
+     onto the second function.
```

```suggestion
  /*
    `copyFunctionArgs f g` creates a new function `g'` with the same behavior as `g` (`g' x == g x`)
    but its function arguments mirroring `f` (`lib.functionArgs g' == lib.functionArgs f`).
```

Since:
- We aren't changing the passed in function, we're creating a new one
- We shouldn't talk about internals (`__functionArgs`)

---

I'd also recommend indenting the entire doc comment by 2 spaces compared to the attribute name to make it nicer on editors.